tests/functional/tests/rgw/test_rgws.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+
+
+class TestRGWs(object):
+
+    @pytest.mark.no_docker
+    def test_ceph_rgw_package_is_installed(self, node, Package):
+        # ceph-radosgw is the package name on centos, radosgw name on ubuntu
+        assert (Package("ceph-radosgw").is_installed or Package("radosgw").is_installed)
```

Comment:
@andrewschoen I thought `is_installed` would evaluate to true/false as described [here](http://testinfra.readthedocs.io/en/latest/modules.html#testinfra.modules.package.Package.is_installedl) but instead it is raising an AssertionError right when `is_installed` evaluates to 1 as seen [here](https://jenkins.ceph.com/job/ceph-ansible-prs-jewel-ansible2.2-xenial_cluster/321/console). Any thoughts on what to do?